docs/reference-units.md

```diff
@@ -95,4 +95,5 @@
     | [Weber](https://en.wikipedia.org/wiki/Weber_(unit)) | `weber`, `Wb` |
     | [Week](https://en.wikipedia.org/wiki/Week) | `weeks`, `week` |
     | [Yard](https://en.wikipedia.org/wiki/Yard) | `yards`, `yard`, `yd` |
-    | [Year](https://en.wikipedia.org/wiki/Year) | `years`, `year` |
+    | [Gregorian year](https://en.wikipedia.org/wiki/Gregorian_year) | `years`, `year` |
+    | [Julian year](https://en.wikipedia.org/wiki/Julian_year_(astronomy)) | `julianYears`, `julianYear` |
```

Comment:
I'm wondering if we should explicitly mention the length of the years in the table. One can easily check the length by going to the Wikipedia pages linked here though.

Also, should we keep the unit name as "Year" instead of "Gregorian year"? I think the latter is clearer.

Reply:
> I'm wondering if we should explicitly mention the length of the years in the table. One can easily check the length by going to the Wikipedia pages linked here though.

Yeah, we don't do anything like this for the other units either.

> Also, should we keep the unit name as "Year" instead of "Gregorian year"?

:+1:

Reply:
Sorry, I'm not exactly clear on what your opinions on the two issues here are. Are you saying the PR is good as-is?

Reply:
Yes, exactly. I like that we have "year" in the documentation. And I think it's okay if we don't mention the precise distinction between the two in the documentation.

Reply:
To be clear, by "unit name" I meant the name in the first column of the table. Also, I made the unit name in the table "Gregorian year", not "year", is that what you want?

Reply:
> To be clear, by "unit name" I meant the name in the first column of the table.

Oh, my bad. I misunderstood your initial post. I'm okay with leaving "Gregorian year" here. Maybe we simply write "(Gregorian) year"?

Reply:
Hmm, I don't think the parentheses are needed, just "Gregorian year" is okay in my opinion.

Reply:
Alright.